src/main/java/se/sundsvall/precheck/integration/partyassets/PartyAssetsIntegration.java

```diff
@@ -0,0 +1,35 @@
+package se.sundsvall.precheck.integration.partyassets;
+
+import generated.client.partyAssets.Asset;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import se.sundsvall.precheck.api.model.Status;
+
+import java.util.List;
+
+@Component
+public class PartyAssetsIntegration {
+
+    static final String INTEGRATION_NAME = "party-assets";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PartyAssetsIntegration.class);
+
+    private final PartyAssetsClient client;
+
+    public PartyAssetsIntegration(final PartyAssetsClient client) {
+        this.client = client;
+    }
+
+    public ResponseEntity<List<Asset>> getPartyAssets(final String a, final Status status) {
+        try {
+            LOGGER.info("Calling PartyAssets service");
+            return client.getPartyAssets(a, status.toString());
+        } catch (Exception e) {
```

Comment:
Should all exceptions be interpreted as "NOT_FOUND".

Example of possible scenarios:
Bad input is sent to the client: NOT_FOUND
Service is down: NOT_FOUND
The client is configured in a bad way: NOT_FOUND

Reply:
Better error handling has been added in both integrations.

Thanks for the comment! 😄